src/doc/reference/glossary/deadlock.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Deadlock
+tags: deadlock
+---
+A situation when two or more {% defn "strand" %}
+instances are each waiting for another to
```

Comment:
Why "strand instances" instead of just "strands"?

Reply:
I don't know :-)
PS: fixed it